docs/document/docs/guides/table-readonly-table-customization.md

```diff
@@ -90,11 +90,11 @@ When you edit the input in the first cell, the value received by the table will
 ## Examples
 ### Basic Usage
 #### ReadOnlyTable
-![readonly-table](assets/readonly-table.png)
+![readonly-table](/img/readonly-table.png)
 
 Display a read-only mode table.
 
-<details>
+<details className="toggle-panel">
   <summary>Show code</summary>
```

Comment:
I think no need border between "Show code" and source code itself.

Reply:
Please pull latest code